src/components/editor/ArticleListController.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import FalcorController from "lib/falcor/FalcorController";
+import List from "./List";
+
+var pageLength = 50;
```

Comment:
Okay yeah.

As of my current design it's supposed to be constant, so yeah I'll do that. Is it clear then if it's constant? Or you still think I should name it something like NUM_ARTICLES_SHOWN?